tests/test_api.py

```diff
@@ -3496,7 +3496,7 @@ def setUp(self):
                            country_code='US')
 
     def test_merge_identities(self):
```

Comment:
If `merge_identities` method is finally renamed to `merge_individuals`, this test should be renamed too.